clusterloader2/testing/experimental/storage/pod-startup/ephemeral-volumes/1_node/config.yaml

```diff
@@ -3,7 +3,7 @@
 
 #Constants
 {{$TOTAL_PODS := DefaultParam .TOTAL_PODS 1}}
-{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "../volume-types/emptydir/pod_with_emptydir.yaml"}}
+{{$DEPLOYMENT_TEMPLATE_PATH := DefaultParam .DEPLOYMENT_TEMPLATE_PATH "../volume-types/emptydir/deployment_with_emptydir.yaml"}}
```

Comment:
I'm sorry - I was blocking @hantaowang PRs for quite some time and your stuff, so this time I will do it the other way (unfortunately you're both touching the same stuff).
https://github.com/kubernetes/perf-tests/pull/688 is super useful, so I would like that one to merge first and then I will ask you to rebase this one on top.